lib/service/Service.ts

```diff
@@ -181,8 +181,12 @@ class Service extends EventEmitter {
    * Cancel placed order from the orderbook.
    */
   public cancelOrder = async ({ orderId, pairId }: { orderId: string, pairId: string }) => {
```

Comment:
Please check the return types and values.

Reply:
What do you mean? I am returning the boolean  `canceled` which is the exact same value as before.

Reply:
@michael1011 never mind i overlooked 👍